gps_uart.c

```diff
@@ -1,6 +1,6 @@
 #include <string.h>
 
-#include <minmea.h>
+#include "minmea.h"
```

Comment:
Can you explain the reasoning here? `minmea.h` is provided via `fap_private_libs` and the submodule under `lib`, so my understanding is it should be included by search path with `<>`. As far as I can tell changing to quotes should have no effect.

```suggestion
#include <minmea.h>
```